manifests/cf-manifest/cloud-config/020-cf-vm-types.yml

```diff
@@ -0,0 +1,125 @@
+---
+meta:
+  default_env:
+    bosh:
+      password: (( grab secrets.vcap_password ))
+
+vm_types:
+  - name: small
+    network: cf
+    env: (( grab meta.default_env ))
+    cloud_properties:
+      instance_type: m3.medium
```

Comment:
Btw. new stemcells are HVM and from previous gen instances, only CC2, CR1, HI1, and HS1 support it. All of which are quite huge instances.